wgpu-hal/src/gles/device.rs

```diff
@@ -858,6 +858,7 @@ impl crate::Device<super::Api> for super::Device {
                 version: self.shared.shading_language_version,
                 writer_flags,
                 binding_map,
+                push_constant_binding: 0, //TODO?
```

Comment:
Yes, I think doing this should work. The backend still isn't exposing PUSH_CONSTANTS feature, so it should fail the validation before using this `0`.